brainscore/benchmarks/rajalingham2019.py

```diff
@@ -1,6 +1,8 @@
 import itertools

```

Comment:
please double-check this file before merging, I think some pieces in here might not be merged correctly with the current file version